### PR TITLE
Add note about reaction emojis

### PR DESCRIPTION
**Pull request by jasonyavorsky**
To quote Madison, who summed it up very well: Reaction emojis are a great way to engage with content in Slack without creating unnecessary noise. Not everything requires a discussion in thread and a simple acknowledgment can signal to the sender that their message reached an active audience.

### CODE REVIEW

**Review by courier-new — APPROVED** [review]
Great idea to capture this explicitly! I've been making this recommendation to friends at other companies, too. I think you summed it up well. Could it be worth mentioning this as an alternative to creating noise, as well? Like if a "thank you" emoji could get the job done, prefer it to actually replying "thank you" since the latter would create an unnecessary notification?

File: content/company-info-and-process/communication/asynchronous-communication.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ Slack is our [team chat](../communication/team_chat.md) and a big part of our da
...
 - A good practice is to send one message with your request directly, instead of sending “[Hey, how’s it going](https://www.nohello.com/)?” and waiting for the other person to reply to send the actual message. That interrupts their work without communicating anything. If you have a quick question, just ask the question directly, and the person will respond asynchronously.
+- Slack can be a black hole. You might leave a message and nobody replied to it - does that mean everyone saw it and agrees? Does it mean that nobody saw it? Or that people saw it and are choosing not to engage? You can help mitigate this by leaving reaction emojis even if you don't plan to reply. A checkmark, thank you, heart, thumbs up, or thumbs down are good ways to indicate you saw it and what you thought about it.
```
Comment (jasonyavorsky) [inline content/company-info-and-process/communication/asynchronous-communication.md]:
```suggestion
- Slack can be a black hole. You might leave a message and nobody replied to it - does that mean everyone saw it and agrees? Does it mean that nobody saw it? Or that people saw it and are choosing not to engage? You can help mitigate this by leaving reaction emojis even if you don't plan to reply. A checkmark, thank you, heart, thumbs up, or thumbs down are good ways to indicate you saw it and what you thought about it. Within a team you can also think of setting more explicit rules for emoji meanings as the [Enablement group has done here](../../departments/product-engineering/engineering/enablement/index.md#slack-acknowledgment).
```

File: content/company-info-and-process/communication/asynchronous-communication.md [hunk 2]
```diff
@@ -112,6 +112,7 @@ Slack is our [team chat](../communication/team_chat.md) and a big part of our da
...
 - A good practice is to send one message with your request directly, instead of sending “[Hey, how’s it going](https://www.nohello.com/)?” and waiting for the other person to reply to send the actual message. That interrupts their work without communicating anything. If you have a quick question, just ask the question directly, and the person will respond asynchronously.
+- Slack can be a black hole. You might leave a message and nobody replied to it - does that mean everyone saw it and agrees? Does it mean that nobody saw it? Or that people saw it and are choosing not to engage? You can help mitigate this by leaving reaction emojis even if you don't plan to reply. A checkmark, thank you, heart, thumbs up, or thumbs down are good ways to indicate you saw it and what you thought about it. Within a team you can also think of setting more explicit rules for emoji meanings as the [Enablement group has done here](../../departments/product-engineering/engineering/enablement/index.md#slack-acknowledgment).
```
Comment (jasonyavorsky) [inline content/company-info-and-process/communication/asynchronous-communication.md]:
```suggestion
- Slack can be a black hole where it feels like messages go out, but it is unclear if anyone saw or engaged with it. Not everything needs a thread, but when nobody replies does that mean everyone saw it and agrees? Does it mean that nobody saw it? Or that people saw it and are choosing not to engage? You can help mitigate this lack of clarity by leaving reaction emojis even if you don't plan to reply. A checkmark, thank you, heart, thumbs up, or thumbs down are good ways to indicate you saw it and what you thought about it. Within a team you can also think of setting more explicit meanings as the [Enablement group has done here](../../departments/product-engineering/engineering/enablement/index.md#slack-acknowledgment).
```

File: content/company-info-and-process/communication/asynchronous-communication.md [hunk 3]
```diff
@@ -112,6 +112,7 @@ Slack is our [team chat](../communication/team_chat.md) and a big part of our da
...
 - A good practice is to send one message with your request directly, instead of sending “[Hey, how’s it going](https://www.nohello.com/)?” and waiting for the other person to reply to send the actual message. That interrupts their work without communicating anything. If you have a quick question, just ask the question directly, and the person will respond asynchronously.
+- Slack can be a black hole where it feels like messages go out, but it is unclear if anyone saw or engaged with it. Not everything needs a thread, but when nobody replies does that mean everyone saw it and agrees? Does it mean that nobody saw it? Or that people saw it and are choosing not to engage? You can help mitigate this lack of clarity by leaving reaction emojis even if you don't plan to reply. A checkmark, thank you, heart, thumbs up, or thumbs down are good ways to indicate you saw it and what you thought about it. Within a team you can also think of setting more explicit meanings as the [Enablement group has done here](../../departments/product-engineering/engineering/enablement/index.md#slack-acknowledgment).
```
Comment (jasonyavorsky) [inline content/company-info-and-process/communication/asynchronous-communication.md]:
```suggestion
- Slack can be a black hole where it feels like messages go out, but it is unclear if anyone saw or engaged with it. Not everything needs a thread, but when nobody replies does that mean everyone saw it and agrees? Does it mean that nobody saw it? Or that people saw it and are choosing not to engage? You can help mitigate this lack of clarity by leaving reaction emojis even if you don't plan to reply. A checkmark, thank you, heart, thumbs up/down, or one of the values emojis (`:value-*`) are good ways to indicate you saw the message and what you thought about it without having to reply. Within a team you can also think of setting more explicit meanings as the [Enablement group has done here](../../departments/product-engineering/engineering/enablement/index.md#slack-acknowledgment).
```

File: content/company-info-and-process/communication/asynchronous-communication.md [hunk 4]
```diff
@@ -112,6 +112,7 @@ Slack is our [team chat](../communication/team_chat.md) and a big part of our da
...
 - A good practice is to send one message with your request directly, instead of sending “[Hey, how’s it going](https://www.nohello.com/)?” and waiting for the other person to reply to send the actual message. That interrupts their work without communicating anything. If you have a quick question, just ask the question directly, and the person will respond asynchronously.
+- Slack can be a black hole where it feels like messages go out, but it is unclear if anyone saw or engaged with it. Not everything needs a thread, but when nobody replies does that mean everyone saw it and agrees? Does it mean that nobody saw it? Or that people saw it and are choosing not to engage? You can help mitigate this lack of clarity by leaving reaction emojis even if you don't plan to reply. A checkmark, thank you, heart, thumbs up/down, or one of the values emojis (`:value-*`) are good ways to indicate you saw the message and what you thought about it without having to reply. Within a team you can also think of setting more explicit meanings as the [Enablement group has done here](../../departments/product-engineering/engineering/enablement/index.md#slack-acknowledgment).
```
Comment (jasonyavorsky) [inline content/company-info-and-process/communication/asynchronous-communication.md]:
```suggestion
- Slack can be a black hole where it feels like messages go out, but it is unclear if anyone saw or engaged with it. Not everything needs a thread, but when nobody replies does that mean everyone saw it and agrees? Does it mean that nobody saw it? Or that people saw it and are choosing not to engage? You can help mitigate this lack of clarity by leaving reaction emojis even if you don't plan to reply. A checkmark, thank you, heart, thumbs up/down, or one of the values emojis (`:value-*:`) are good ways to indicate you saw the message and what you thought about it without having to reply. Within a team you can also think of setting more explicit meanings as the [Enablement group has done here](../../departments/product-engineering/engineering/enablement/index.md#slack-acknowledgment).
```

**Pull request merged**